frontend/src/components/Pages/RobotPage/RobotPage.tsx

```diff
@@ -166,7 +166,9 @@ export const RobotPage = ({ robotId }: { robotId: string }) => {
                                 <StyledLeftContent>
                                     <RobotImage height="350px" robotType={selectedRobot.model.type} />
                                     {stopButton}
-                                    {selectedRobot && <ReturnHomeButton robot={selectedRobot} />}
+                                    {selectedRobot && selectedRobot.status != RobotStatus.InterventionNeeded && (
+                                        <ReturnHomeButton robot={selectedRobot} />
+                                    )}
```

Comment:
Besides this change, maybe you can merge your two commits? Currently running only the first one will result in a frontend warning